x-pack/platform/plugins/shared/onechat/common/http_api/agents.ts

```diff
@@ -5,7 +5,23 @@
  * 2.0.
  */
 
+import type { AgentProfile } from '@kbn/onechat-common';
+
 export interface CallAgentResponse<TResult = unknown> {
   runId: string;
   result: TResult;
 }
+
+export type GetAgentResponse = AgentProfile;
+
+export interface ListAgentsResponse {
+  agentProfiles: AgentProfile[];
```

Comment:
could we start calling them agents now? Have that prop across all naming, removing `profile`.